frontend/src/component/PublicProfile/ProfileMainBar.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {
+  Avatar,
+  AVATAR_SIZE,
+  Breadcrumbs
+} from 'tracim_frontend_lib'
+import PropTypes from 'prop-types'
+
+export const ProfileMainBar = props => {
+  const hasUser = Object.keys(props.displayedUser).length !== 0
```

Comment:
Is it possible that this component receives an empty user?

Reply:
Yes, while the component did not receive the response from the API